pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -203,9 +206,78 @@ func (a *weeklyAverageFromTenDays) getAggregatedTestRuns(ctx context.Context) (m
 	return a.aggregatedTestRunsByName, a.queryTestRunsErr
 }
 
+func getMajor(in string) int {
```

Comment:
Hashicorp go-version is vendored here. Would it work for your needs?

Reply:
Good suggestion. I will look at this tomorrow to see if it works. I stole the code from origin. :)

Reply:
It's not a big deal either way, if we use this code in other places it's fine too

Reply:
go-version does not manipulate version number. So it does not do what we want here.